packages/asc-ui/src/components/Accordion/Accordion.stories.mdx

```diff
@@ -4,6 +4,7 @@ import { withKnobs, boolean, text } from '@storybook/addon-knobs'
 import { Meta, Story, Preview } from '@storybook/addon-docs/blocks'
 import Accordion from './Accordion'
 import AccordionWrapper from './AccordionWrapper'
+import Paragraph from '../Paragraph'
```

Comment:
Why import Paragraph here?

Reply:
It's used futher down, but the import was missing. Caused some errors to show up in the console.